build/index.js

```diff
@@ -110,6 +112,24 @@ examples.forEach(example => {
   console.log('saved to file examples/' + fileName)
 })
 
+// Write revelant playground fixtures to a Jekyll collection
+fs.emptyDir(fixturesCollection, err => {
+  axe.getRules().forEach(rule => { // only fixtures with a corresponding axe rule
+    const filename = path.join(root, './playground', 'fixtures', rule.ruleId) + '.html'
+    fs.readFile(filename, 'utf8', (err, data) => {
+      if (!err && data && data.length) {
+        frontLoad(
+          path.join(root, fixturesCollection, rule.ruleId) + '.html',
```

Comment:
I don't think that's much of an advantage. If something is outdated, we shouldn't keep it around. If the goal is to prevent us from publishing fixtures that don't have rules to go with it, shouldn't the build step just delete those fixtures for us instead? Either that, or shout at us for not cleaning it up ourselves.